lib/datadog/appsec/contrib/rack/request_middleware.rb

```diff
@@ -57,6 +57,10 @@ def call(env)
 
             AppSec::Event.record(*context.events)
 
+            if response_action && response_action.any? { |action, _event| action == :block }
```

Comment:
Looks good to me, I think this is the behavior change

Reply:
Yes that is 😄